### DISCUSSION TITLE
Support for Danfo.js in Arquero.js as a backend?

### DISCUSSION THREAD

**Discussion by javierluraschi** [opensource9ja/danfojs#286] [Ideas]
Related to https://twitter.com/risingodegua/status/1436387929371389957 and https://news.hal9.ai/posts/data-science-with-javascript

Basically, we found out [Arquero.js](https://github.com/uwdata/arquero) is pretty great at doing transformations and working against a database; however, it does not play well with TF.js but it does support backends... so it should be possible to build a Danfo.js backend for Arquero.js.

We would be interested to help, so let us know!

**Comment by risenW** [comment]
Thanks @javierluraschi I’ll go through Arquery over this weekend. By then I would have enough background to contribute to this discussion.

cc @steveoni

**Comment by steveoni** [comment]
nice one @risenW  @javierluraschi  I will also look into this.
